Resources/gulp/scripts/_ckdatatable.js

```diff
@@ -31,7 +31,9 @@
                 this.initTable();
             }
             else{
-                this.table.ajax.reload();
+                this.table.ajax.reload(() => {
```

Comment:
I think you need to rebuild js files
![image](https://github.com/CrossKnowledge/DataTableBundle/assets/237869/8043f033-de90-4149-96a0-78649bbebaa9)